README.md

```diff
@@ -1,13 +1,32 @@
-# Northcoders News API
 
----
+# Northcoders News API 
 
-1) SETUP LOCAL DATABASES - Create two .env files: .env.test and .env.development. Add PGDATABASE=nc_news_test to the test file and PGDATABASE=nc_news to the development file. Ensure that these .env files are .gitignored.
 
-2) CREATE DATABASES: run the db/setup.sql file: npm run setup-dbs.
+--- NC News --- 
 
-3) SEED DEVELOPMENT DATABASE: run the db/run-seed.js file: npm run seed.
+DESCRIPTION - a web service for displaying and updating Northcoders’ news updates, including articles, topics, comments and votes. 
 
---- 
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+1) CLONE REPOSITORY - on Github: https://github.com/GadgetGeekette/be-nc-news: Fork > Create a new fork > Create fork. From the Code dropdown copy the HTTPS URL. In a terminal enter: git clone <forked_repo_url>
+
+2) INSTALL DEPENDENCES - install add dependencies using the CLI command: npm i 
```

Comment:
It may be nice to add some formatting. e.g. you could add backticks around npm i to make it look like `npm i`